src/sidebar/util/test/sentry-test.js

```diff
@@ -100,8 +94,8 @@ describe('sidebar/util/sentry', () => {
       );
     });
 
-    it('disables the URL allowlist if `document.currentScript` is inaccessible', () => {
-      fakeDocumentCurrentScript.get(() => null);
+    it('disables the URL allowlist if the script URL is unavailable', () => {
+      import.meta.url = null;
```

Comment:
Is it `null` for sandboxed iframes?

Reply:
The URL refers to the script rather than the document, so it won't be null in a sandboxed iframe. Re-reading the [relevant spec](https://html.spec.whatwg.org/multipage/webappapis.html#hostgetimportmetaproperties) it doesn't look like the URL _can_ be falsey. It is possible for `URL(import.meta.url).origin` to be the string "null" if the script is an inline script and the document URL is `about:blank`. This script is always loaded externally though, so that shouldn't happen.

Reply:
I suppose some third-party JS on the page could manipulate this URL, as we found is possible to do in the tests.